### ISSUE TITLE
Contributions Form

### ISSUE THREAD

**Issue by DanGould** [citizensgov/citizensgov2020#13]
Largely copy [Stripe Payments Demo](https://stripe-payments-demo.appspot.com/) via [Stripe elements](url) and [payments request API](https://stripe.com/docs/stripe-js/elements/payment-request-button#set-up-stripe-elements) to support payment via debit, credit, and Payment Request API associated products (i.e. apple pay and the like).

In addition to card info which stripe collects, we personally need to collect
- Full Name
- Address, zip code
- Employer / self employed / not employed
- occupation

@verquist If possible, it would be nice to avoid collecting all of this information for every donor since law doesn't require reporting it for donations YTD < $200, but I'm not sure if there's a way around it. What are your thoughts?

**Comment by verquist** [comment]
My understanding is that we need to capture all the information for every donor. I researched this and all political donation software seems to work this way. I also investigated other political campaigns. Even though reporting requirements stipulate that we only need to break out donors separately when their donations total >= $200, I believe we need to capture and retain this information in our records for every donor, as proof that we honored campaign finance laws.

**Comment by verquist** [comment]
We may want to model our notices and disclaimers on Elizabeth Warren’s donation page: https://secure.actblue.com/donate/ew-homepage-hero

**Comment by DanGould** [comment]
Good suggestion. We ought to get a privacy policy together as well before accepting donations

**Comment by verquist** [comment]
Yes, we need to create a privacy policy before accepting donations. I’ll add it to my todo list. Thank you!
